hack/make-rules/test-integration.sh

```diff
@@ -95,7 +95,7 @@ runTests() {
       WHAT="${WHAT:-$(kube::test::find_integration_test_pkgs | paste -sd' ' -)}" \
       GOFLAGS="${GOFLAGS:-}" \
       KUBE_TIMEOUT="${KUBE_TIMEOUT}" \
-      KUBE_RACE="" \
```

Comment:
>   # KUBE_RACE and MAKEFLAGS are used in the downstream make, and we set them to
  # empty here to ensure that we aren't unintentionally consuming them from the
  # previous make invocation.

is this ok despite the comment?